core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts

```diff
@@ -593,15 +775,28 @@ export class CreateServiceComponent implements OnInit {
     }
 
     /* Including deployment_accounts in the payload */
-    let deployment_accounts = [
-      {
-        "accountId": this.accountSelected,
-        "region": this.regionSelected,
-        "provider":"aws",
-        "primary":true
-      }
-    ]
-    payload['deployment_accounts'] = deployment_accounts
+    if(this.typeOfPlatform === 'aws'){
+      let deployment_accounts = [
+        {
+          "accountId": this.accountSelected,
+          "region": this.regionSelected,
+          "provider": this.typeOfPlatform,
+          "primary":true
+        }
+      ]
+      payload['deployment_accounts'] = deployment_accounts
+    }
+    else if(this.typeOfPlatform === 'azure') {
+      let deployment_accounts = [
```

Comment:
Do these change? If not, prefer `const`. Lint will address in the future.

Reply:
Done. Changed to const.